### PR TITLE
Add package parameter for legacy support

### PR DESCRIPTION
**Pull request by brechtm**
Not yet tested, but the change seems trivial enough.

### CODE REVIEW

**Comment by brechtm** [comment]
I tried to test this, but I can't seem to get choco to install from a local source.

File: automatic/imagemagick.app/tools/chocolateyInstall.ps1 [hunk 1]
```diff
@@ -24,6 +24,9 @@ if ($env:chocolateyPackageParameters) {
     if ($packageParams.InstallDevelopmentHeaders) {
         $packageArgs.silentArgs = $packageArgs.silentArgs + ' /MERGETASKS=install_devel'
     }
+    if ($packageParams.LegacySupport) {
+        $packageArgs.silentArgs = $packageArgs.silentArgs + ' /MERGETASKS=legacy_support'
```
Comment (bdukes) [inline automatic/imagemagick.app/tools/chocolateyInstall.ps1]: It looks like when there are multiple tasks, they need to be comma separated (i.e. `/MERGETASKS=install_devel,legacy_support` if both were selected).

**Comment by bdukes** [comment]
Thanks, looks to me like this is working.  I appreciate you making that change.

In the future, you should be able to test by, first, creating a package using `cpack`, and then installing by setting the source to the current directory (e.g. `choco install imagemagick.app -Source . -PackageParameters LegacySupport=true`)

Thanks!

File: automatic/imagemagick.app/tools/chocolateyInstall.ps1 (hunk 1 above)
Comment (brechtm) [inline automatic/imagemagick.app/tools/chocolateyInstall.ps1]: It is possible to pass /MERGETASKS several times.

**Comment by brechtm** [comment]
Thanks for the quick review, merge and cpack hint. Cheers!

File: automatic/imagemagick.app/tools/chocolateyInstall.ps1 (hunk 1 above)
Comment (bdukes) [inline automatic/imagemagick.app/tools/chocolateyInstall.ps1]: When I call `ImageMagick-7.0.3-4-Q16-x64-dll.exe /MERGETASKS=install_devel /MERGETASKS=legacy_support` it only checks the Legacy Support checkbox, but not the Development Headers checkbox

Comment (brechtm) [inline automatic/imagemagick.app/tools/chocolateyInstall.ps1]: Uh. I'm sure I tested that, but I must not have been very focused... good thing you checked.

**Pull request merged**